### ISSUE TITLE
fix typos in Spanish WG/WS

### ISSUE THREAD

**Issue by vmarchman** [langcog/web-cdi#305]
@HenryMehta I have given you the updated files for Spanish WG, but I would need the files for Spanish WS.

**Comment by HenryMehta** [comment]
@vmarchman 
Does this mean the WG files didn't get loaded to CATCDI?

**Comment by vmarchman** [comment]
Hi @HenryMehta .  Ok, the Spanish WG fixes are in dev, but not prod.  Can we move them to prod?

**Comment by vmarchman** [comment]
Can you send me the files for Spanish WS so I can make those fixes?

**Comment by HenryMehta** [comment]
@vmarchman 

WG deployed

I think this is all the files
[Spanish_WS.zip](https://github.com/langcog/web-cdi/files/5540366/Spanish_WS.zip)

**Comment by HenryMehta** [comment]
@vmarchman 
Errors seem to be in the _meta.json files.

WG:  I could not find "Eso lo sabemos responde a la palabras y frases" but found "Eso lo sabemos responde a palabras y frases" (no la) and changed to "Eso lo sabemos responde a las palabras y frases".

WS: I changed "en las lista" to "en la lista".  I also changed "Aquí nos interesan la palabras que " to "Aquí nos interesan las palabras que"

Deployed and ready to test
